src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -84,6 +84,14 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
     return e2eiService;
   }
 
+  private createOIDCService() {
```

Comment:
Shouldn't we check if this.oidcService already exists and return that instance if available?

Reply:
Well, I replaced the `new OIDC` to this new function. 
There might be some problems if we start reusing old instances when we used to create new ones. 

So I went to the safe side an re-created new instances everytime.